packages/expo-location/src/Location.types.ts

```diff
@@ -211,6 +211,12 @@ export type LocationTaskServiceOptions = {
    * Boolean value whether to destroy the foreground service if the app is killed.
    */
   killServiceOnDestroy?: boolean;
+  /**
+   * Boolean value whether to show the notification in the status bar. Defaults to false.
```

Comment:
```suggestion
   * Boolean value whether to show the notification in the status bar.
```

This is redundant, when using `@default` tag.